modules/nf-core/zip/tests/main.nf.test.snap

```diff
@@ -0,0 +1,10 @@
+{
+    "versions": {
+        "content": [
+            [
+                "versions.yml:md5,721b9a1e22ba1d75fe6a30bb1684cf02"
+            ]
+        ],
```

Comment:
you are not capturing anything else other than versions in the snapshot ?

Reply:
I am checking [here](https://github.com/nf-core/modules/blob/8debddd2876a7e079f552fbbae7d01b97911fdc1/modules/nf-core/zip/tests/main.nf.test#L27) that the zip file is produced as the file md5 hash changes. I have seen this in other modules, e.g. https://github.com/nf-core/modules/blob/master/modules/nf-core/salmon/quant/tests/main.nf.test.snap do I have to do something else @sateeshperi ? Thanks!

Reply:
if there is nothing to be done...its fine